atlas/templates/cmd/server/profiler.go.gotmpl

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	_ "net/http/pprof"
+)
+
+const (
+	//Profiler
+	defaultProfilerEnabled = false
+	defaultProfilerAddress = "0.0.0.0"
+	defaultProfilerPort    = "9999"
+)
+
+var (
+	flagProfilerEnabled = pflag.Bool("profiler.enable", defaultProfilerEnabled, "enable profiler by default false")
+	flagProfilerAddress = pflag.String("profiler.address", defaultProfilerAddress, "address of profiler server")
+	flagProfilerPort    = pflag.String("profiler.port", defaultProfilerPort, "port of profiler server")
```

Comment:
From my point of view it's better to move configuration specific code into config.go file to be consistent